3dText_client.lua

```diff
@@ -0,0 +1,32 @@
+--CLIENT
+local drawDistance = 150
+local playerVisibleTextEntities = {}
+
+function streaming()
+	local px,py,pz = getElementPosition(getLocalPlayer())
+	triggerServerEvent("onClientRequest3dText", root, px,py,pz,10,150)
```

Comment:
Skoro to i tak stałe to nie ma sensu ich przekazywać w evencie tylko zrobić na sztywno consta serverside

Reply:
Nie będzie stałą prawdopodobnie, trzymam to w pamięci żeby wrócić do tego